src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -89,7 +90,10 @@ public void sendMail(String host, String from, String to, String subject, String
             String[] recipientStrings = to.split(",");
             InternetAddress[] recipients = new InternetAddress[recipientStrings.length];
             try {
-                msg.setFrom(new InternetAddress(from, charset));
+            	InternetAddress fromAddress=getSystemAddress();
+            	fromAddress.setPersonal(fromAddress.getPersonal() + "on behalf of " + reply, charset);
```

Comment:
Should "on behalf of" be put into Bundle.properties so it can be translated into various languages?

Reply:
See `BundleUtil.getStringFromBundle` examples in this file (lower down).